dist/cydia/deb_hand.mak

```diff
@@ -126,4 +126,4 @@ clobber::
 		${PACKAGE_DIR}/data ${PACKAGE_DIR}/build
 
 push:
-	scp *.deb radare.org:/srv/http/radareorg/cydia/debs
+	scp *.deb radare.org:/srv/https/radareorg/cydia/debs
```

Comment:
that change is not correct. the internal path in the server is http, not https

Reply:
ok